src/components/deals/loading/LoadingPreview.tsx

```diff
@@ -1,11 +1,65 @@
+import Section, { SECTION_STYLE, SECTION_WIDTH } from '@/components/Section'
 import LoadingDealImage from './LoadingDealImage'
-
+import LoadingTagsList from './LoadingTagsList'
+import LoadingText, {
+  LOADING_TEXT_STYLE,
+  LOADING_TEXT_WIDHTS,
+} from './LoadingText'
```

Comment:
I find a line break between sections helpful (like imports and exports).

It feels cramped

Reply:
Yep, that's fair. I typically do. Don't know why not here. Added in latest commit!